Sources/SwiftDocC/Infrastructure/Topic Graph/AutomaticCuration.swift

```diff
@@ -134,7 +154,7 @@ public struct AutomaticCuration {
         {
             // Group match in render context, verify if there are any other references besides the current one.
             guard linkingGroup.references.count > 1 else { return nil }
```

Comment:
Nit since this is a little out-of-scope, but the existing logic seems a little hacky to me and would break if we ever stop including the current reference in the linking group.

```suggestion
            let references = try filterReferences(linkingGroup.references)
            guard !references.isEmpty else { return nil }
```

Reply:
The automatic curation logic here finds a task group that contains this reference, removing this reference from the task group. The `count > 1` is essentially to not emit the task group at all if the only reference it contains is this reference.

Reply:
Right I see that- I'm just suggesting we do the filtering ahead of time so we can gate on the already filtered group being empty or not.